public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -267,7 +263,7 @@ const EpicTestEditor: React.FC<EpicTestEditorProps> = ({
         </div>
       )}
 
-      {!isOffPlatform && test.variants.length > 0 && (
+      {epicEditorConfig.allowCustomVariantSplit && test.variants.length > 1 && (
```

Comment:
I think we might want `> 1` here as that means we have a control + at least one variant

Reply:
Agreed